src/Arcus.Templates.ServiceBus.Topic/Program.cs

```diff
@@ -13,20 +13,14 @@ namespace Arcus.Templates.ServiceBus.Topic
 {
     public class Program
     {
-#if (ExcludeSerilog == false)
+#if Serilog
         #warning Make sure that the appsettings.json is updated with your Azure Application Insights instrumentation key.
         private const string ApplicationInsightsInstrumentationKeyName = "TELEMETRY_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY";
```

Comment:
Same here, the setting should be called `APPINSIGHTS_INSTRUMENTATIONKEY`

Reply:
Done in https://github.com/arcus-azure/arcus.templates/pull/587/commits/844d1b8d5f71c36dc7c34c097bf47629a2088bd1